packages/markdown-editor/src/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -55,6 +60,11 @@ const editorConfig = {
   ],
 };
 
+export const MarkdownEditorContext = React.createContext<MarkdownEditorContextValue>({
```

Comment:
there is no reason to use a Context for this implementation, we are just passing props to a children component